### DISCUSSION TITLE
Sometimes github copilot doesn't load solutions, any reason why this happens?

### DISCUSSION THREAD

**Discussion by supermavster** [community/community#18863] [Copilot Conversations]
I use GitHub copilot a lot when I'm developing, which is an excellent tool.

But sometimes, the sample of solutions stops; I restart the IDE or the code editor, and still no response.

It rarely happens, any explanation of what is happening or any solution?

**Comment by akshay-99h** [comment]
It is usually due to data connectivity problems. A change in network connection can help.
Do check if you have enabled copilot for the specific file extension.

If you tried the above solutions and the issue persists, it might just be the copilot figuring out the best suggestion for your code and looking for further input from the coder's end.

**Comment by davecheney** [comment]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.
